recipes/delly/meta.yaml

```diff
@@ -1,15 +1,20 @@
 package:
   name: delly
-  version: '0.7.2'
+  version: '0.7.6'
 
 source:
-  git_url: https://github.com/tobiasrausch/delly.git
-  git_rev: v0.7.2
+  fn: delly_v0.7.6_linux_x86_64bit
+  url: https://github.com/tobiasrausch/delly/releases/download/v0.7.6/delly_v0.7.6_linux_x86_64bit
+  md5: 148ed6dde55f07491138226e53198a47
 
 build:
   number: 1
   skip: True # [osx]
 
+requirements:
+  build:
+  run:
+
 test:
   commands:
     - delly --help 2>&1 | grep delly
```

Comment:
Since you are the dev of delly: does it already exit with exitcode 0 upon --help? If that is the case, the grepping can be removed and we can just do the following:
`delly --help 2> /dev/null`

Also, providing help output at stdout if imho better, because of piping into e.g. less.

If not, that's all fine for this release, but I would suggest those changes for the next, just to be more standards compliant.